Jubjubnest.Style.DotNet/Resources.resx

```diff
@@ -462,4 +462,10 @@
   <data name="NameReadOnlyFieldsWithCamelCase_Title" xml:space="preserve">
     <value>Read-only fields should be named with camelCasing</value>
   </data>
+  <data name="BaseConstructorCallSameLine_Message" xml:space="preserve">
+    <value>Move the constructor call to its right place.</value>
+  </data>
+  <data name="BaseConstructorCallSameLine_Title" xml:space="preserve">
+    <value>Base constructor call should be on the same line as the last closing brace of the parameter list if the parameters are on their own lines. Otherwise it should be on its own line.</value>
+  </data>

```

Comment:
Couple of errors in the strings:

- Message: "Move the __constructor__ call to its right place." - should probably be "base constructor".
- Title: "last closing __brace__" - should be "parenthesis" (`)` opposed to `}`)

And some general notes on the resources.

The `Title` is shown in the name column in the ruleset designer. For this reason something shorter that can be displayed easily without a "..." in the end would be better. If needed, the `Description` resource can be used for more detailed information. This is displayed in the ruleset editor when the specific rule is highlighted.

![image](https://user-images.githubusercontent.com/385385/34047030-a6a73680-e1b7-11e7-8eee-eb43645ab845.png)

Also the `Message` should include some details of what item this concerns. This is mainly for the benefit when going through the "Error List" window content. Although in this specific case the fact that it's `constructor` in a specific file should be enough in most cases.